packages/node_modules/@cerebral/http/src/HttpProviderError.js

```diff
@@ -1,11 +1,11 @@
 export default class HttpProviderError extends Error {
-  constructor(status, headers, body, message = null, isAborted = false) {
+  constructor(status, headers, result, message = null, isAborted = false) {
     super()
     this.name = 'HttpProviderError'
     this.message = message
     this.status = status
     this.headers = headers
-    this.body = body
+    this.result = result
```

Comment:
Yeah, we can make it the same as the output of the `response` on operators. That would make things consistent.

So:

```js
    this.name = 'HttpProviderError'
    this.message = message // responseText
    this.response = {
      status, headers, result
    }
```

Looks good?